easybuild/easyblocks/t/tensorflow.py

```diff
@@ -257,6 +257,8 @@ def configure_step(self):
         apply_regex_substitutions('configure.py', regex_subs)
 
         cmd = self.cfg['preconfigopts'] + './configure ' + self.cfg['configopts']
+        # Tell Bazel to not use $HOME/.cache/bazel at all
+        env.setvar('TEST_TMPDIR', os.path.join(tmpdir, 'output_root'))
```

Comment:
@akesandgren Can you move this above the `cmd =`?

Is there a reference to the Bazel documentation that mentions the use of `$TEST_TMPDIR`?

Reply:
Yes, there is, site/docs/output_directories.md

Reply:
No URL to include? :-/